alpha/action/render.go

```diff
@@ -237,6 +247,19 @@ func (r Render) imageToDeclcfg(ctx context.Context, imageRef string) (*declcfg.D
 	return cfg, nil
 }
 
+func (r *Render) renderFBCDirectory(ctx context.Context, dir string) (*declcfg.DeclarativeConfig, error) {
+	// If a filterer is provided that can tell us which meta objects to keep, we
+	// can optimize FBC loading and post-filtering by ignoring meta objects that
+	// we know we won't need to keep.
+	keepMeta := func(meta *declcfg.Meta) bool { return true }
+	if r.Filterer != nil {
+		if metaFilterer, ok := r.Filterer.(filter.MetaFilterer); ok {
+			keepMeta = metaFilterer.KeepMeta
+		}
+	}
```

Comment:
Nit: Why not have an explicit field for the `MetaFilterer`?

Reply:
At the time, I think it was because I was actually implementing inlined functions with that same signature, so it was lighter weight than implementing a full struct.

I'll add a function type that implements the interface (sorta like http.Handler and http.HandlerFunc).

Reply:
I'm also thinking about names... the age old problem.